pyomyo.py

```diff
@@ -43,6 +43,8 @@
 import sys
```

Comment:
yo fix this

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix